plugins/protocolbuffers/pyi/v25.3/buf.plugin.yaml

```diff
@@ -14,5 +14,5 @@ registry:
     requires_python: ">=3.8"
     deps:
       # https://pypi.org/project/protobuf/
-      - "protobuf"
+      - "protobuf~=4.25"
```

Comment:
I've pulled this change in https://github.com/bufbuild/plugins/pull/1287/commits/f33db4d8e03ad630d2188c1fada476ba08c2162d which will hopefully land today (it has some dependencies on the CLI to get a green build). I'd prefer to do this together so we only bump the revision once.